src/test/java/wooteco/subway/path/domain/SubwayPathTest.java

```diff
@@ -0,0 +1,100 @@
+        SectionEdge sectionEdge2 = new SectionEdge(new Section(station2, station3, 25), line);
+
+        subwayPath = new SubwayPath(Arrays.asList(sectionEdge1, sectionEdge2),
+            Arrays.asList(station1, station2, station3), 0);
+        assertThat(subwayPath.calculateFareByDistance()).isEqualTo(2050);
+        assertThat(subwayPath.getDistance()).isEqualTo(50);
+        assertThat(subwayPath.getStations()).isEqualTo(Arrays.asList(station1, station2, station3));
+    }
+
+    @DisplayName("50키로 초과인 경우")
+    @Test
+    void extraDistanceFare() {
+        SectionEdge sectionEdge1 = new SectionEdge(new Section(station1, station2, 30), line);
+        SectionEdge sectionEdge2 = new SectionEdge(new Section(station2, station3, 30), line);
+
+        subwayPath = new SubwayPath(Arrays.asList(sectionEdge1, sectionEdge2),
+            Arrays.asList(station1, station2, station3), 0);
+        assertThat(subwayPath.calculateFareByDistance()).isEqualTo(2450);
+        assertThat(subwayPath.getDistance()).isEqualTo(60);
+        assertThat(subwayPath.getStations()).isEqualTo(Arrays.asList(station1, station2, station3));
+    }
+
+    @DisplayName("환승이 없는 경우에 요금이 추가되는 경우")
+    @Test
+    void oneLineExtraFare() {
+        Section section1 = new Section(station1, station2, 30);
+        Section section2 = new Section(station2, station3, 30);
+        Sections sections = new Sections(Arrays.asList(section1, section2));
+        line = new Line(1L, "testLine1", "black", sections, 900);
+
+        SectionEdge sectionEdge1 = new SectionEdge(section1, line);
+        SectionEdge sectionEdge2 = new SectionEdge(section2, line);
+
+        subwayPath = new SubwayPath(Arrays.asList(sectionEdge1, sectionEdge2),
+            Arrays.asList(station1, station2, station3), 0);
+        assertThat(subwayPath.calculateFareByDistance()).isEqualTo(3350);
+    }
+
+    @DisplayName("환승이 있는 경우에 요금이 추가되는 경우")
+    @Test
+    void twoLineExtraFare() {
+        Section section1 = new Section(station1, station2, 30);
+        Section section2 = new Section(station2, station3, 30);
+        Sections sections1 = new Sections(Collections.singletonList(section1));
+        Sections sections2 = new Sections(Collections.singletonList(section2));
+        line = new Line(1L, "testLine1", "black", sections1, 900);
+        Line line1 = new Line(2L , "testLine2", "white", sections2, 1000);
+        SectionEdge sectionEdge1 = new SectionEdge(section1, line);
+        SectionEdge sectionEdge2 = new SectionEdge(section2, line1);
+
+        subwayPath = new SubwayPath(Arrays.asList(sectionEdge1, sectionEdge2),
+            Arrays.asList(station1, station2, station3), 0);
+        assertThat(subwayPath.calculateFareByDistance()).isEqualTo(3450);
+        assertThat(subwayPath.getDistance()).isEqualTo(60);
+        assertThat(subwayPath.getStations()).isEqualTo(Arrays.asList(station1, station2, station3));
```

Comment:
```suggestion
        assertThat(subwayPath.getStations()).containsExactly(station1, station2, station3);
```

테스트 메서드가 너무 길고 가독성이 떨어지는 모습이 보이는데요, 어떻게 개선할 수 있을지 고민해 보시면 좋을 것 같아요. 테스트 코드도 계속 관리해야 하는 대상임을 항상 고려해 주세요.

Reply:
최대한 리팩터링을 진행해보았는데 아직까지 맘에 안드는 부분이 존재하네요ㅠㅠ 더 개선하려고 노력하겠습니다😥